rustfmt.toml

```diff
@@ -64,7 +64,7 @@ use_field_init_shorthand = false
 force_explicit_abi = true
 condense_wildcard_suffixes = false
 color = "Auto"
-required_version = "1.5.2"
+required_version = "1.5.3"
```

Comment:
I don't remember why we added this but it's been annoying to update for each release so lets remove it. According to the [docs](https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#required_version), it's mostly good for CI

Reply:
I rebased my branch on top of the main so you have resolved it for me :)